app/Http/Controllers/PetController.php

```diff
@@ -5,20 +5,24 @@
 use App\Models\Pet;
 use Illuminate\Http\Request;
 
-class PetController extends Controller
-{
-    public function show() {
-        return view('pets', [
-            'pets' => self::getPet(),
-        ]);
-        return null;
+class PetController extends Controller  {
+    public function show(){
+        return view (
+            'pets',
+            [
+                'pets' => self::getPet(),
+            ]);
+            return null;
+
     }
 
     public static function getPet() {
```

Comment:
FYI this can be private scope since it's only called within the class definition. Also if it's returning an array, I'd name it `getPets`